src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -193,14 +193,14 @@ private void AnalyzeDirectory(DirectoryInfo directoryPath, ReportFormat reportFo
                 if (!directoryPath.Exists)
                 {
                     logger.LogError("Invalid directory: {directoryPath}", directoryPath);
-                    return;
+                    return 2;
```

Comment:
Define constants for the exit codes at the top of the file and return those instead of raw numbers.

Reply:
sounds good